src/types.ts

```diff
@@ -102,6 +103,7 @@ export type TInternalConfig = {
   logoutRedirect?: string
   preLogin?: () => void
   postLogin?: () => void
+  loginMethod?: 'redirect' | 'popup'
```

Comment:
```suggestion
  loginMethod: 'redirect' | 'popup'
```
Since it has a default value, no need to make it optional (possibly null)

Reply:
Thanks! Fixed it.